ambry-replication/src/main/java/com/github/ambry/replication/ReplicaThread.java

```diff
@@ -1629,7 +1630,7 @@ synchronized boolean isEmpty() {
     }
 
     /**
-     *  get the keys corresponding to the missing messages in local store
+     *  Get the keys corresponding to the missing messages in local store
      */
     Set<StoreKey> getMissingStoreKeys() {
       return missingStoreMessages == null ? Collections.emptySet()
```

Comment:
I was under the impression the stream way to collect to set is deep copy, so this should be ok (better to confirm it)